controllers/ingestion/http/http.go

```diff
@@ -0,0 +1,79 @@
+package http
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+)
+
+// DruidHTTP interface
+type DruidHTTP interface {
```

Comment:
Is this interface necessary? Are we going to add more clients?

Reply:
idk yet.just keeping it clean